WORKSPACE

```diff
@@ -2,13 +2,20 @@ workspace(name = "com_github_istio_pilot")
 
 git_repository(
     name = "io_bazel_rules_go",
-    commit = "eba68677493422112dd25f6a0b4bbdb02387e5a4",  # Aug 1, 2017
+    commit = "0e5a0e51b4e9fc3b5ef1436639a43fce27559744",  # Oct 3, 2017
     remote = "https://github.com/bazelbuild/rules_go.git",
 )
 
-load("@io_bazel_rules_go//go:def.bzl", "go_repositories", "go_repository")
+git_repository(
+    name = "com_github_google_protobuf",
+    commit = "593e917c176b5bc5aafa57bf9f6030d749d91cd5",  # Jan 2017 3.2.0
+    remote = "https://github.com/google/protobuf.git",
+)
+
+load("@io_bazel_rules_go//go:def.bzl", "go_rules_dependencies", "go_repository", "go_register_toolchains")
 
-go_repositories()
+go_rules_dependencies()
+go_register_toolchains()
```

Comment:
nit: put go version here, it defaults to 1.9.1 right now.

Reply:
how do you mean? a comment?

Reply:
Yeah, `go_version=blah`. I think we do need to move over to 1.9.1 (which is the implicit default here).

Reply:
oh oh. are we staying at go1.8.4 for now?

Reply:
I don't see a good reason to stay on old version in pilot.